pkgs/development/libraries/libxkbcommon/default.nix

```diff
@@ -1,7 +1,8 @@
 { stdenv, fetchurl, fetchpatch, meson, ninja, pkgconfig, yacc, xkeyboard_config, libxcb, libX11, doxygen }:
 
 stdenv.mkDerivation rec {
-  name = "libxkbcommon-0.8.0";
+  name = "libxkbcommon-${version}";
+  version = "0.8.0";
```

Comment:
this is causing the mass-rebuild.

Reply:
If you undo it, we can merge it into master.

Reply:
OK

Reply:
DONE :+1: